src/ReactRouter/ReactRouter.tsx

```diff
@@ -0,0 +1,26 @@
+
+import Landingpage from '../LandingPage/LandingPage';
+import Roadmap from '../Roadmap/Roadmap';
+import orgChart from '../orgChart/OrgChartTree';
+import Introduction from '../LandingPage/Introduction';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+
+
+function ReactRouter() {
+    return (
+        <Router>
+            <div>
+                <Switch>
+                    <Route path="/" exact component={Landingpage}/>
+                    <Route path="/yo'l-xaritasi"  component={Roadmap} />
+                    <Route path="/azolar-daraxti"  component={orgChart} />
+                    <Route path="/batafsil-malumot"  component={Introduction} />
```

Comment:
Odatda kodda keladigan hamma narsa ingliz tilida yoziladi. Bizdagi o‘, g‘ kabi xarflar ayrim joylarda muammo tug‘dirganligi sababli shunisi yaxshiroq.
```suggestion
                    <Route path="/roadmap"  component={Roadmap} />
                    <Route path="/organization-tree"  component={orgChart} />
                    <Route path="/introduction"  component={Introduction} />
```